internal/helm/chart/builder.go

```diff
@@ -81,9 +81,9 @@ func (r RemoteReference) Validate() error {
 	if r.Name == "" {
 		return fmt.Errorf("no name set for remote chart reference")
 	}
-	name := regexp.MustCompile("^([-a-z0-9]*)$")
+	name := regexp.MustCompile("^([-a-z0-9/]*)$")
```

Comment:
To block things such as `/` and `abc//cba`:

```suggestion
	name := regexp.MustCompile("^([-a-z0-9]+/?)+$")
```